File: landsfairwell/go/ReCall.go
package main

import (
	"fmt"
)

func main() {

	fmt.Println("Hello World")
}

func findGreaterNumber(a, b int) int {
	if a > b {
		return a
	} else {
		return b
	}
}

func getMonthNameByNumber(a int) string {
	switch a {
	case 1:
		return "January"
	case 2:
		return "Feb"
	case 3:
		return "March"
	case 4:
		return "April"
	case 5:
		return "May"
	case 6:
		return "June"
	case 7:
		return "July"
	case 8:
		return "August"
	default:
		return "Sorry! invalid number"
	}
}

func LoopIngTest() {
	for i := 0; i < 10; i++ {
		fmt.Println(i)
	}
}

File: landsfairwell/python/Recall.py
#variables
#Control Flow
##Function

def findGreaterNumber(a,b):
	if a>b:
		return a
	else:
		return b
	

def getMonthNameByNumber(a):
	match a:
		case 1:
			return "January"
		case 2:
			return "Feb"
		case 3:
			return "March"
		case _:
			return "Sorry Invalid Number"

def LoopingTest():
	i=0
	while i<10:
		print(i)
		i=i+1
		
	
def main():
    print("Hello World")
    greaterNumber = findGreaterNumber(10,15)
    print(greaterNumber)
    LoopingTest()
    getMonthNameByNumber(1)


if __name__ == "__main__":
    main()




